packages/features/ee/teams/components/AddNewTeamMembers.tsx

```diff
@@ -37,7 +33,7 @@ const AddNewTeamMembers = () => {
   const session = useSession();
   const teamId = searchParams?.get("id") ? Number(searchParams.get("id")) : -1;
   const teamQuery = trpc.viewer.teams.get.useQuery(
-    { teamId },
+    { teamId, isOrg: !!session.data?.user.organizationId },
```

Comment:
Letting the team getter to return team even if the logged in user does not belong to it

Reply:
Do we need to send this as input? Don't we have this session in the server request?

Reply:
@leog i don't see isOrg in input schema of viewer.team.get